### ISSUE TITLE
Each page should have a unique title

### ISSUE THREAD

**Issue by erinchambers** [CDRH/austen#79]
Two steps here:
1. The `<title>` in the HTML `<head>` of the page should be unique from page to page. A good general structure is `Page Title | Site Title`, e.g. `Word Frequencies | Austen Said`.
2. The `<h1>` for the page should be the page title, not the site title in the header.

**Comment by wkdewey** [comment]
I made a solution, but then (becuase I initially looked at `main` rather than `dev`) I saw that an attempt had already been made to fix this in layouts/application.erb.html in `dev`. My work overwrites that. When I tested `dev` the existing solution did not seem to work.

**Comment by wkdewey** [comment]
I did not address the `<h1>` issue. Do you want to overwrite the header? (h1 on the header might break an accessibility guideline, but surely we don't want to get rid of the header or change it to the page title. I think the header is usually an image) Or add an additional h1 lower on the page?

**Comment by erinchambers** [comment]
I can take it from here! Thanks, Will!

**Comment by erinchambers** [comment]
Fixed by #88
